core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -331,13 +331,35 @@ angular.module('oppia').component('contributionsAndReview', {
         }
       };
 
+      ctrl.getActiveDropdownTabChoice = function() {
+        if (ctrl.activeTabType === ctrl.TAB_TYPE_REVIEWS) {
+          if (ctrl.activeSuggestionType === SUGGESTION_TYPE_QUESTION) {
+            return 'Review Questions';
+          } else {
```

Comment:
Don't need the `else` clause, can extract the return statement. Ditto for the elses below.

Reply:
Removed the `else`s.